test/src/test_derive.rs

```diff
@@ -8,6 +8,13 @@ pub(crate) fn run_tests() -> bool {
 
     status &= test_derive_to_variant();
     status &= test_derive_owned_to_variant();
+    status &= test_derive_nativeclass();
+    status &= test_derive_nativeclass_omitted_inherit();
+    status &= test_derive_nativeclass_omitted_base_parameter();
```

Comment:
Maybe use "without" instead of "omitted", so it's consistent with `test_derive_nativeclass_without_constructor` below.

Reply:
Fixed. Reflected in next push.